webhook/webhook.go

```diff
@@ -147,6 +152,8 @@ func New(
 	logger := logging.FromContext(ctx)
 
 	if opts.StatsReporter == nil {
+		// Register webhook metrics
+		metricsOnce.Do(func() { RegisterMetrics(opts.StatsReporterOptions...) })
```

Comment:
Before `RegisterMetrics` would always be called.  Now it's only being invokved when the `opts.StatsReporter` isnt' set.

This seems like a breaking change

Reply:
I agree with your assessment. It might break users who specify `StatsReporter` yet still rely on `RegisterMetrics` to register views properly.

However I consider this as an unpleasant situation when users want to use a different `StatsReporter` but register the same view with slightly different tags.

To be fair, this is not something that I need anymore since the default StatsReporter can be customized already.

As it is today, `webhook.NewStatsReporter()` and `webhook.RegisterMetrics()` seems to be independent interfaces while they actually share the same data structure (tracks the same metrics tags, and relies on the same `opts.StatsReporterOptions` to initialize).

Reply:
@skonto do you have any opinions?

Reply:
I think a migration path would be to have a flag `disableDefaultMetrics` with default being `false`.